californium-core/src/main/java/org/eclipse/californium/core/network/stack/BlockwiseLayer.java

```diff
@@ -115,8 +115,8 @@
 	private final LeastRecentlyUsedCache<KeyUri, Block1BlockwiseStatus> block1Transfers;
 	private final LeastRecentlyUsedCache<KeyUri, Block2BlockwiseStatus> block2Transfers;
 	private int maxMessageSize;
-	private int preferredBlockSize;
-	private int preferredBlockSzx;
+	protected int preferredBlockSize;
+	protected int preferredBlockSzx;
```

Comment:
I wouldn't change this to `protected` just for adjust them to BERT values. Instead, I would add the adjustment to the CoapTcpStack and correct the `config` there. I would also only create a `TcpBlockwiseLayer` instead of a `BlockwiseLayer`, if enabled.